CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+* **2017-02-03**: [BC break] Removed own SimplePager in favour of `Sonata\AdminBundle\Datagrid\SimplePager` to avoid duplications.
```

Comment:
You're not really supposed to touch this, that's what the changelog in the PR body is about. Please remove this and make the changelog PR as good as this one.